web/src/stream_popover.js

```diff
@@ -75,7 +75,7 @@ export function topic_sidebar_menu_handle_keyboard(key) {
     popovers.popover_items_handle_keyboard(key, items);
 }
 
-function elem_to_stream_id($elem) {
+export function elem_to_stream_id($elem) {
```

Comment:
The commit message "Export ...", not "Extract ..." -- "Extract" would usually be used to refer to taking a block of code and making it a function with name.

Reply:
Here's how I rewrote this -- other detail that I think is worth highlighting is that "the migration" is ambiguous for someone who's looking at the commit history, not just this commit, so you should always be specific in what you're talking about.

```
popovers: Export stream_popover.elem_to_stream_id.                 
                                                                   
This is a prep commit for migrating the topic popover to a Tippy popover.  
                                                                   
'elem_to_stream_id' function is used for the topic menu to find the
stream_id of the target topic menu.                                
```

Reply:
Sure, thanks a lot for the suggestion making the change.